### ISSUE TITLE
Setters for flags should accept params

### ISSUE THREAD

**Issue by WittleWolfie** [WittleWolfie/WW-Blueprint-Core#45]
Right now flags accept arrays but don't use params syntax:

```
SetFlag(FlagEnum[] values)
```

Should be

```
SetFlag(params FlagEnum[] values)
```

**Comment by WittleWolfie** [comment]
Maybe more than just that.... I don't think any of the setters work right damn.

**Comment by WittleWolfie** [comment]
Fix is in 2.0.x
